rules/shared/builtin-errors.js

```diff
@@ -7,7 +7,6 @@ const builtinErrors = [
 	'SyntaxError',
 	'TypeError',
 	'URIError',
-	'InternalError',
```

Comment:
This is non-standard https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/InternalError